cmd/common-main.go

```diff
@@ -105,6 +105,7 @@ func init() {
 	gob.Register(StorageErr(""))
 	gob.Register(madmin.TimeInfo{})
 	gob.Register(madmin.XFSErrorConfigs{})
+	gob.Register(madmin.SysConfig{})
```

Comment:
```suggestion
	gob.Register(map[string]string{})
```

The error is coming because of `"thp-config"` which is set as `map[string]string{}`